collector/prometheus.go

```diff
@@ -0,0 +1,57 @@
+package collector
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+)
+
+type Label struct {
+	LabelName  []string
+	LabelValue []string
+}
+
+type PromCounter struct {
```

Comment:
```suggestion
type promCounter struct {
```
I think we should hide the underlying counter type to external users